sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/CosmosError.java

```diff
@@ -68,7 +65,7 @@ public CosmosError(String errorCode, String message, String additionalErrorInfo)
      * @return the error code.
      */
     public String getCode() {
-        return this.jsonSerializable.getString(Constants.Properties.CODE);
+        return this.getString(Constants.Properties.CODE);
```

Comment:
Though it will not make a difference for this class, but the correct implementation should be `super.set / get`
Instead of `this.getString()`, all of the APIs inside this class should be calling `super.getString()`
Here and below for other APIs - should be -> `super.get*()` and `super.set*()`

Reply:
changed to use super.